src/platform_impl/linux/menu.rs

```diff
@@ -176,6 +198,145 @@ impl Menu {
     menu
   }
 
+  #[cfg(feature = "tray")]
+  pub fn to_ksnimenu(
+    &self,
+    tx: &Sender<(WindowId, WindowRequest)>,
+    window_id: WindowId,
+  ) -> super::system_tray::KsniMenu {
+    let mut v = vec![];
+    for item in &self.gtk_items {
+      if let Some(m) = Self::generate_ksnimenu_item(item, tx.clone(), window_id) {
+        v.push(m);
+      }
+    }
+    v
+  }
+
+  #[cfg(feature = "tray")]
+  fn generate_ksnimenu_item(
+    item: &GtkMenuInfo,
+    tx: Sender<(WindowId, WindowRequest)>,
+    window_id: WindowId,
+  ) -> Option<super::system_tray::KsniMenuItem> {
+    match item {
+      GtkMenuInfo {
+        menu_type: GtkMenuType::Custom,
+        sub_menu: None,
+        custom_menu_item: Some(i),
```

Comment:
I don't like one letter variables
```suggestion
        custom_menu_item: Some(item),
```

Reply:
Variable has been renamed to `custom_item` in e6cff2b. This name has been chosen to prevent any further confusion with the `item` variable that is created earlier.